src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -647,62 +723,65 @@ function FilterSidebarSearchTab({
+                            }
+                            disabled={fetchingOptions}
+                            fullWidth
                         >
-                            Report an issue
-                        </a>
-                    ) : null}
-                    <div className="offset offset-right" style={{ flex: 1 }}>
+                            Search
+                        </Button>
+                    )}
+                    {checkIfAnyFieldSelected(allFields) ? (
                         <Button
                             size="small"
                             variant="outlined"
                             onClick={() => resetFilters(embed)}
                             disableRipple
-                            className={classes.font}
+                            className={classes.reset}
                             color="primary"
                             disabled={fetchingOptions}
                         >
-                            Reset
+                            <i
+                                className={`${classes.icon} fas fa-fw fa-undo`}
+                            />
                         </Button>
-                        {fetchingFacilities ? (
-                            <CircularProgress
-                                size={30}
-                                className="margin-left-16"
+                    ) : null}
+                </div>
+                <div className="form__field">
+                    {expandButton}
+                    <ShowOnly when={!expand}>
+                        <div className="form__info">
+                            Contributor type · Parent company · Facility type ·
+                            Processing type · Product type · Number of workers{' '}
+                            <i
+                                className={`${classes.icon} fas fa-fw fa-info-circle`}
                             />
-                        ) : (
-                            <Button
-                                variant="contained"
-                                size="small"
-                                type="submit"
-                                color="primary"
-                                className={`margin-left-16 ${classes.font}`}
-                                style={{
-                                    boxShadow: 'none',
-                                }}
-                                onClick={() =>
-                                    searchForFacilities(vectorTileFlagIsActive)
-                                }
-                                disabled={fetchingOptions}
-                            >
-                                Search
-                            </Button>
-                        )}
-                    </div>
+                        </div>
+                    </ShowOnly>
+                </div>
+                <div className="form__report">
```

Comment:
I want to make this link sticky to the bottom and also scrollable with the search tab. I tried different ways but not none of them works as intended

Reply:
I'm having trouble picturing what you're aiming for. Can you describe it a bit more?

Reply:
I'm trying to this "Have a question?" link stay at the bottom of this section but not blocking any other contents while scrolling. I tried to use `position: absolute` but that results in the text "floating" on other components when scrolling
<img width="352" alt="image" src="https://user-images.githubusercontent.com/60887686/158431381-1db5fd16-8408-4d93-b95a-52333480dde0.png">